descriptors/ModuleDescriptor-template.json

```diff
@@ -16,6 +16,15 @@
       "id": "data-import",
       "version": "3.0",
       "handlers": [
+      	{
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/data-import/jobexecutions/{jobExecutionId}/cancel",
```

Comment:
seems to me that this endpoint should be `DELETE /data-import/jobexecutions/{jobExecutionId}` or `POST /data-import/jobexecutions/{jobExecutionId}/cancel`. What do you think?

Reply:
OR `PUT /data-import/jobexecutions/{jobExecutionId}/cancel`

Reply:
Leaning towards the `POST /cancel`, going to confer with FE

Reply:
We're going to keep it the same for consistency with previous endpoints used in mod-srm.